components/centraldashboard/public/components/manage-users-view.pug

```diff
@@ -0,0 +1,35 @@
+h1 Manage Contributors
+iron-ajax#AddContribAjax(method='POST', url='/api/add-contributor', on-response='handleContribCreate', on-error='handleContribCreate')
```

Comment:
This method doesn't exist on the server.

Reply:
Fixed, outdated